payments_service/braintree/templates/braintree/emails/subscription_charged_successfully.html

```diff
@@ -0,0 +1,13 @@
+{# subscription header is added automatically here #}
+    <table>
+        <tr><th>Date</th><td>{{ date }}</td></tr>
+        <tr><th>Receipt #</th><td>{{ transaction.uuid }}</td></tr>
+        <tr><th>Paid with</th><td><div class="card-icon cctype-{{ cc_type|lower }}"></div> ending in {{ cc_truncated_id }}</td></tr>
+        <tr><th>&nbsp;</th><td></td></tr>
```

Comment:
Probably should use CSS for this e.g. add more padding to the description row or something.